ezflow/engine/trainer.py

```diff
@@ -2,16 +2,27 @@
 from copy import deepcopy
 
 import torch
+import torch.backends.cudnn as cudnn
+import torch.distributed as dist
+import torch.multiprocessing as mp
 import torch.nn as nn
 from torch.nn.parallel import DistributedDataParallel as DDP
 from torch.utils.tensorboard import SummaryWriter
 
-from ..data import DeviceDataLoader
 from ..functional import FUNCTIONAL_REGISTRY
 from ..utils import AverageMeter, endpointerror
 from .registry import loss_functions, optimizers, schedulers
 
 
+def seed(seed):
+
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+    cudnn.benchmark = False
+    cudnn.deterministic = True
```

Comment:
I think it would be better to set the seed for NumPy as well. 
`np.random.seed(seed)`